pkg/steps/write_params.go

```diff
@@ -55,15 +55,15 @@ func (s *writeParametersStep) run(dry bool) error {
 }
 
 func (s *writeParametersStep) Requires() []api.StepLink {
-	return s.params.AllLinks()
+	return []api.StepLink{api.AllStepsLink()}
```

Comment:
I have no idea WTF this step is, if it's ever actually used, or what. This is technically a change in behavior but should be safe?

Reply:
Yeah I think this is a :clayton: